aodncore/pipeline/destpath.py

```diff
@@ -86,13 +86,14 @@ def _resolve_from_handler(self):
     def _resolve_from_param(self):
         if isinstance(self.function_param, six.string_types):
             # a string parameter is assumed to be referring to an advertised entry point of the same name
+            loaded_functions, failed_functions = self.handler_instance.config.discovered_dest_path_functions
```

Comment:
Should there be at least a warning if there are any `failed_functions`?

Reply:
This class is only concerned with a single *current* function, so doesn't make sense to warn about unrelated functions. I have however updated the exception message to differentiate between missing and failed functions

Reply:
:+1: